bind/motor-mobile/bind.go

```diff
@@ -16,15 +16,15 @@ var (
 	instance    mtr.MotorNode
 )
 
-func Init(buf []byte, cb ct.MotorCallback) ([]byte, error) {
+func Init(buf []byte) ([]byte, error) {
```

Comment:
This change would break Motor Bind @joshLong145 please dont include this change

Reply:
I just noticed this after build the framework. i reverted that change